Moved the commands to appropriate modules.

boczkowska opened the pull request:
Moved a few commands to appropriate modules.

Addresses these two issues:
- https://github.com/openshift/openshift-docs/issues/20136
- https://github.com/openshift/openshift-docs/issues/20134

Branches 4.2, 4.3, 4.4

kalexand-rh commented on modules/openshift-developer-cli-creating-and-deploying-a-nodejs-application-with-odo.adoc:
```diff
@@ -8,10 +8,45 @@
 .Procedure 
 
-. Change the current directory to the front-end directory:
+ifeval::["{context}" == "creating-a-single-component-application-with-odo"]
```
We've switched to using the ifeval statements to set variables based on the context statements and then just use if statements to conditionalize the text, like here: https://raw.githubusercontent.com/openshift/openshift-docs/master/modules/installation-initializing.adoc

boczkowska replied:
Oh, I missed that change! Sorry, fixed. The build looks fine, so, hopefully, I've done in correctly. 
What was the reasoning for switching to variables?

kalexand-rh replied:
You can't put two arguments in an ifeval statement, so if you need to use the same text in multiple locations, you have to duplicate it. It's a good way to make it easier to reuse content as you use it in more locations.

boczkowska replied:
Thank you! That makes total sense.

kalexand-rh commented on modules/openshift-developer-cli-creating-and-deploying-a-nodejs-application-with-odo.adoc:
```diff
@@ -8,10 +8,45 @@
-. Change the current directory to the front-end directory:
+ifeval::["{context}" == "creating-a-single-component-application-with-odo"]
+
+. Create a new directory for your components:
```
~new~

kalexand-rh commented on modules/openshift-developer-cli-creating-and-deploying-a-nodejs-application-with-odo.adoc:
```diff
@@ -8,10 +8,45 @@
 
```
You need a short context statement that describes the purpose of the steps. All procedure modules need to have them.

boczkowska replied:
I've added introductions for modules that this PR touched. Some other modules might miss intros, I'll scope and update them separately so this PR won't blow out of proportion.

kalexand-rh replied:
I think it's a good improvement to make as you touch modules or when you get space to work on RFEs.

kalexand-rh commented on modules/openshift-developer-cli-deploying-the-back-end-component.adoc:
```diff
@@ -31,10 +31,22 @@ nodejs        openshift     10,8,8-RHOAR,latest
 openjdk18     myproject     latest
 ----
 
+. Create a new directory for your components:
```
~new~

kalexand-rh commented on modules/openshift-developer-cli-creating-a-project-in-odo.adoc:
```diff
@@ -15,48 +15,10 @@
-
-endif::[]
-
 . Create a new project:
```
~new~

kalexand-rh commented on modules/openshift-developer-cli-creating-and-deploying-a-nodejs-application-with-odo.adoc:
```diff
@@ -8,10 +8,45 @@
+----
+endif::[]
+
+ifeval::["{context}" == "creating-a-multicomponent-application-with-odo"]
```
Please update the list of assemblies that use this module.

kalexand-rh commented on modules/openshift-developer-cli-creating-and-deploying-a-nodejs-application-with-odo.adoc:
```diff
@@ -8,10 +8,45 @@
+. Download the example back-end and front-end applications:
++
+----
+$ git clone https://github.com/openshift-evangelists/Wild-West-Frontend frontend
```
Does support know that you have instructions to clone content from github?

boczkowska replied:
Yes, we've approved all of our external URLs with GSS.

boczkowska commented:
@kalexand-rh thank you for your feedback. I've addressed all of the issues you pointed out and also squashed the commits. Please take a look when you have time.

kalexand-rh commented on modules/openshift-developer-cli-deploying-the-front-end-component.adoc:
```diff
@@ -3,12 +3,27 @@
+
 [id="deploying-the-front-end-component_{context}"]
 
+To create and deploy a front-end component, download the Node.js application and push the source code to your cluster with `{odo-title}`.
```
@boczkowska, please put this after the heading.

boczkowska replied:
Oops, sorry about that. Fixed.

kalexand-rh commented:
/cherrypick enterprise-4.2

kalexand-rh commented:
/cherrypick enterprise-4.3

kalexand-rh commented:
/cherrypick enterprise-4.4
